server/sonar-server/src/main/java/org/sonar/server/project/Visibility.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2017 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+package org.sonar.server.project;
+
+import static java.util.Arrays.stream;
+
+public enum Visibility {
+
+  PRIVATE(true, "private"),
+  PUBLIC(false, "public");
+
+  private final boolean isPrivate;
+  private final String word;
+
+  Visibility(boolean isPrivate, String word) {
+    this.isPrivate = isPrivate;
+    this.word = word;
+  }
+
+  public String asWord() {
+    return word;
+  }
+
+  boolean asBoolean() {
+    return isPrivate;
+  }
+
+  public static String asWord(boolean isPrivate) {
```

Comment:
the stream seems overkill here. `return isPrivate ? PRIVATE.getLabel() : PUBLIC.getLabel()`